ntcore/src/main/native/cpp/net/WireEncoder.cpp

```diff
@@ -22,95 +22,58 @@ using namespace mpack;
 void nt::net::WireEncodePublish(wpi::raw_ostream& os, int64_t pubuid,
                                 std::string_view name, std::string_view typeStr,
                                 const wpi::json& properties) {
-  wpi::json::serializer s{os, ' ', 0};
-  os << "{\"method\":\"" << PublishMsg::kMethodStr << "\",\"params\":{";
-  os << "\"name\":\"";
-  s.dump_escaped(name, false);
-  os << "\",\"properties\":";
-  s.dump(properties, false, false, 0, 0);
-  os << ",\"pubuid\":";
-  s.dump_integer(pubuid);
-  os << ",\"type\":\"";
-  s.dump_escaped(typeStr, false);
-  os << "\"}}";
+  wpi::json json = {{"method", PublishMsg::kMethodStr},
+                    {"params",
+                     {{"name", name},
+                      {"properties", properties},
+                      {"pubuid", pubuid},
+                      {"type", typeStr}}}};
+  os << json.dump();
```

Comment:
This (and other changes to WireEncoder) is likely a significant pessimization in what is a relatively common operation.  This creates two `std::map`'s and makes at least 6 allocations.  The old code made zero allocations.